modules/express/src/clientRoutes.ts

```
@@ -352,6 +352,15 @@ function handleCanonicalAddress(req: express.Request) {
   return (coin as Coin.Bch | Coin.Bsv | Coin.Ltc).canonicalAddress(address, version || fallbackVersion);
 }
 
+function handleV1Sign(req: express.Request) {
+  console.log('v1 sign was called');
+  return 'v1_sign_transaction';
+}
+function handleV2Sign(req: express.Request) {
+  console.log('v2 sign was called');
```

same here, let's throw an error for now

---

done